docs/developers/content-creation/creating-a-weapon.md

```diff
@@ -1,3 +1,120 @@
 # Creating a Weapon
 
-Some Text
+When starting content creation for gmod, creating a basic weapon using the SWEP structure is one of the easiest things you can do.
+For this guide I'll be using the Visual Studio Code with the glualint extension as referenced in the "Creating an addon" section of these documents.
+
+## File Format
+
+Firstly you need to make sure that you're using the correct folder structure.
+Your addon folder must be in this format:
+
+* `weapon_ttt_<weapon name>\gamemodes\terrortown\entities\weapons\weapon_ttt_\<weapon name>.lua`
```

Comment:
the addon folder doesn't need to have this name, I'd replace this and the following lines with `<addon_name>\gamemodes\terrortown\entities\weapons\weapon_ttt_\<weapon name>.lua`

Also we are using forward slashes in our documentation

Reply:
Sorry, Windows habits again

Reply:
No need to feel sorry. Your work is great!

Reply:
Awh many thanks